deploy/basic/terraform/network.tf

```diff
@@ -37,7 +37,8 @@ resource "oci_core_subnet" "mushop_lb_subnet" {
   cidr_block                 = lookup(var.network_cidrs, (var.create_secondary_vcn ? "LB-SUBNET-REGIONAL-CIDR" : "MAIN-LB-SUBNET-REGIONAL-CIDR"))
   display_name               = "mushop-lb-${random_string.deploy_id.result}"
   dns_label                  = "mushoplb${random_string.deploy_id.result}"
-  security_list_ids          = [oci_core_security_list.mushop_lb_security_list.id]
+  # If WAF is to be used, do not use the security list - instead the NSG will be applied
+  security_list_ids          = var.enable_waf ? null : [oci_core_security_list.mushop_lb_security_list.id]
```

Comment:
Previous comment about conditionals. The Always Free tier needs to be accounted for, as MuShop basic is primarily targeted at Always Free Tier.